draft-zsf.md

```diff
@@ -0,0 +1,154 @@
+4. **Reduce "Governance Attack Surface":** The proposal's policies aim to enhance predictability, financial sustainability, and minimize governance capture risks, ensuring Zcash's successful evolution as a global currency with a growing userbase and stakeholdership.
+5. **Ecosystem Benefits of Longer Time Horizons:** A reliable and long-term functioning Zcash blockchain allows users to make secure long-term plans, leading to a sustainable and virtuous adoption cycle, rather than being influenced by short-term trends.
+6. **Consensus Continuity During Lulls:** Rate-limited payouts from the Sustainability Fund help maintain blockchain security during periods of low activity by utilizing savings from high activity periods, whereas direct miner rewards tied to short-term activity may favor miners with significant savings reserves or access to credit, reducing mining competitiveness and overall ecosystem certainty about mining capacity.
+7. **Mitigate Payment-to-Self Vulnerabilities:** The risk of miners benefiting from "spoofing" transactions and the desire for a fixed eventual supply goal lead to considering alternatives like the Sustainability Fund, which pays out fees over a long enough time horizon to prevent recouping costs, allowing Zcash to remain closer to its supply goal and offering an alternative to other protocol designs that destroy or burn funds.
+8. **Recovery of "Soft-Burned" Funds:** In some instances, such as miners not claim all available rewards, some ZEC goes unaccounted for though not formally burned. This proposal would recover it through the `ZSF_BALANCE` mechanism described below.
+
+_Disclaimer 1: Long term success depends on the specific mechanisms of deposits, the quantities involved, and broader economic considerations. For such as if the payout rate is both sufficient and not excessive enough to threaten sustainability._
+
+# Specification
+
+In practice, The Zcash Sustainability Fund is a single global balance maintained by the node state and contributed to via a single transaction field. This provides the economic and security support described in the motivation section, while also importantly keeping the fund payouts extremely simple to describe and implement.
+
+The two modifications are:
+1. The re-accounting of unmined ZEC as a node state field called `ZSF_BALANCE`
+2. The addition of a transaction field called `ZSF_DEPOSIT`
+
+Please note that a **network upgrade is required** for this work to be fully implemented.
+
+## `ZSF_BALANCE`
+
+Upon activation at height `H`, the ZEC issuance mechanism is permanently suspended, the value of `ZSF_BALANCE[H]` is set to the remainder of unissued ZEC. This must be done before the `Block Rewards Payout Rule` is enforced for the block at height `H`. **This is not a pre-mine.** This is simply a re-accounting of unissued ZEC that comes with the benefit of other possible sources of funding.
```

Comment:
We believe this ZIP should not change issuance at all.

Reply:
```suggestion
The ZEC issuance mechanism is re-defined to remove funds from `ZSF_BALANCE`, which is initially set to `MAX_MONEY`.
```